README.md

```diff
@@ -105,6 +105,23 @@ There also are some unit tests, you can run those with cargo:
 $ cargo test --all
 ```
 
+# Profiling
```

Comment:
Could you create a seperate document, maybe `development.md` with more in detail information for developers? I already moved the wasm part to a seperate document to keep the first readme file manageable.